### PR TITLE
Cf 896 init script

### PR DESCRIPTION
**Pull request by usnavi**
Adding the following to repose-flume installation:
- Adding /etc/init.d/flume-ng & /etc/sysconfig/flume-ng init scripts for rpm install
- Modifying log4j.properties to write logs to /var/log/flume
- Creating /var/log/flume directory

### CODE REVIEW

File: flume-package/build.gradle [hunk 1]
```diff
@@ -67,15 +82,38 @@ ospackage {
...
+}
+
+buildRpm {
+
+    from( 'src/main/resources/sysconfig' ) {
+        into '/etc/sysconfig'
+        include 'flume-ng'
+        user 'root'
+        permissionGroup 'root'
+        fileMode 0744
```
Comment (shintasmith) [inline flume-package/build.gradle]: This is a weird permission. When I look at the files in our /etc/sysconfig, they are mostly 644.

**Comment by usnavi** [comment]
Thanks @shintasmith  - updated the permissions.

**Comment by shintasmith** [comment]
rest is lgtm

**Comment by gregswift** [comment]
While this works, I would say its generally better to place all of the configs in the init, but after they are assigned and before you validate them source in the sysconfig.  This helps prevent config management from accidentally removing something it doesnt care about but the init script does.

its also pretty common design pattern

**Comment by usnavi** [comment]
gregswift - updated the sysconfig as per our conversation.

**Comment by usnavi** [comment]
@gregswift - pinging you to get the "looks good".  I think everything is in order.

**Comment by gregswift** [comment]
lgtm

**Pull request merged**